certification/internal/policy/operator/scorecard_basic_check_spec.go

```diff
@@ -47,6 +50,13 @@ func (p *ScorecardBasicSpecCheck) Metadata() certification.Metadata {
 }
 
 func (p *ScorecardBasicSpecCheck) Help() certification.HelpText {
+	if p.fatalError {
+		return certification.HelpText{
+			Message: "There was a fatal error while running operator-sdk scorecard tests. " +
+				"Please see the preflight log for details. If necessary, set logging to be more verbose.",
+			Suggestion: "If the logs are showing a context timeout, try setting wait time to a higher value.",
```

Comment:
I think we would need to expose `PFLT_SCORECARD_WAIT_TIME` before we can say this in the help. Right now the value is internally hardcoded.